src/wallet/rpcwallet.cpp

```diff
@@ -101,12 +102,10 @@ std::shared_ptr<CWallet> GetWalletForJSONRPCRequest(const JSONRPCRequest& reques
     }
 
     std::vector<std::shared_ptr<CWallet>> wallets = GetWallets();
-    if (wallets.size() == 1 || (request.fHelp && wallets.size() > 0)) {
+    if (wallets.size() == 1) {
         return wallets[0];
     }
 
-    if (request.fHelp) return nullptr;
-
     if (!HasWallets()) {
```

Comment:
`if (if wallets.size() == 0)` and ditch `HasWallets`.

Reply:
This has one ACK and two stale ACKs, so I'd prefer to keep it as is for now. Maybe @ryanofsky can combine this into the other vpwallets cleaunup?

Reply:
I can open a PR but it's too damn height. No rush really.

Reply:
I am happy to review if you open a pull request

Reply:
Ok #19261